internal/log/log_test.go

```diff
@@ -2,365 +2,56 @@ package log
 
 import (
 	"bytes"
-	"errors"
-	"fmt"
-	"regexp"
-	"strings"
+	"context"
+	"log/slog"
 	"testing"
 
-	"go.uber.org/zap"
-	"go.uber.org/zap/zapcore"
-	"go.uber.org/zap/zaptest/observer"
+	"github.com/stretchr/testify/require"
```

Comment:
I haven't used this package before. Is it worth adding as a dev dependency?

Reply:
Force of habit really. I enjoy it. I don’t have to worry about building test helpers in every package over again. But I understand if we want to stay vanilla. 👍